wiki/Game_mode/osu!taiko/en.md

```diff
@@ -82,137 +82,16 @@ If [Relax](/wiki/Game_modifier/Relax) mod was used, the score judgement only acc
 
 ## Scoring
 
-*Scoring values can be found in [osu!taiko scoring system](/wiki/Score/osu!taiko).*
+[The osu!taiko score](/wiki/Gameplay/Score/osu!taiko) is a weighted sum of all hit objects' values. It depends on the following:
 
-Scoring section details all the intricacies of scoring, including mathematical formula.
+- [Judgement](/wiki/Gameplay/Judgement/osu!taiko) determines an object's base scoring value (300, 100, or 0 in case of a miss). The value of regular and large notes depends on hit timing; every other hit object's base value is constant.
```

Comment:
```suggestion
- [Judgement](/wiki/Gameplay/Judgement/osu!taiko) determines a hit object's base scoring value (300, 100, or 0 in case of a miss). The value of regular and large notes depends on hit timing, while every other object's base value is constant.
```